src/partition/repart.c

```diff
@@ -7909,7 +7909,7 @@ static int find_root(Context *context) {
                         if (!s)
                                 return log_oom();
 
-                        fd = open(arg_node, O_RDONLY|O_CREAT|O_EXCL|O_CLOEXEC|O_NOFOLLOW, 0666);
+                        fd = xopenat_full(AT_FDCWD, arg_node, O_RDONLY|O_CREAT|O_EXCL|O_CLOEXEC|O_NOFOLLOW, XO_NOCOW, 0666);
```

Comment:
i kinda sense that sooner or later someone will show up who wants this configurable.... (env var?)

Reply:
(but this is not a request to add an env var, it's just a general comment)